pom.xml

```diff
@@ -171,6 +181,57 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-install-plugin</artifactId>
 				<version>3.1.1</version>
+				<executions>
+					<!-- Install main JAR -->
+					<execution>
+						<id>install-main-jar</id>
+						<phase>install</phase>
+						<goals>
+							<goal>install-file</goal>
+						</goals>
+						<configuration>
+							<file>${project.build.directory}/${project.build.finalName}.jar</file>
+							<groupId>${project.groupId}</groupId>
+							<artifactId>${project.artifactId}</artifactId>
+							<version>${project.version}</version>
+							<packaging>jar</packaging>
+						</configuration>
+					</execution>
+
+					<!-- Install Javadoc JAR -->
```

Comment:
same here

Reply:
Addressed in [e5a7f84a664b8e666a8cd00b5e15224a6946da2d](https://github.com/ls1intum/Ares2/pull/25/commits/e5a7f84a664b8e666a8cd00b5e15224a6946da2d)